cli/src/main.rs

```diff
@@ -71,27 +73,30 @@ struct Opt {
     command: Command,
 }
 
-fn decrypt(opt: EncryptDecryptOpt) -> Result<()> {
+fn decrypt(opt: EncryptDecryptOpt, cmac: &mut CMac) -> Result<()> {
```

Comment:
How about this? `fn decrypt(opt: EncryptDecryptOpt) -> Result<CMac>`

Reply:
Thanks. Fixed!